### DISCUSSION TITLE
Can dbeaver run on the command line instead of desktop?

### DISCUSSION THREAD

**Discussion by yangmeng66** [dbeaver/dbeaver#20629] [Q&A]
Scenario: Not all linux has a graphical interface to operate dbeaver. What if you want to run dbeaver without a graphical interface?

I found that there are these profiles, what is the difference between these?
![image](https://github.com/dbeaver/dbeaver/assets/51857567/6b38659e-88f2-400c-9b29-97dce86d9ae1)

**Comment by E1izabeth** [comment]
We have a command line tool, but I'm not sure it's possible to not run DBeaver in a GUI at all. However, you might try and tell us :)
See https://dbeaver.com/docs/wiki/Command-Line/

The profiles on the picture is for building and publishing DBeaver. I suppose the profile names speak for themselves.
